lib/ex_subtil_backend/workflow/step/clean_workspace.ex

```diff
@@ -0,0 +1,70 @@
+defmodule ExSubtilBackend.Workflow.Step.CleanWorkspace do
+
+  alias ExSubtilBackend.Jobs
+  alias ExSubtilBackend.Amqp.JobCleanEmitter
+  alias ExSubtilBackend.Workflow.Step.Requirements
+
+  def launch(workflow) do
+
+    paths = get_source_files(workflow.jobs, [])
```

Comment:
the folder pat is generated using:
```elixir
Application.get_env(:ex_subtil_backend, :workdir) / workflow.reference
```
it can be enough to retrieve folder name.

Reply:
But what about the  `dash/` directory? It may be changed into the GPAC worker